pom.xml

```diff
@@ -1093,6 +1093,7 @@ limitations under the License.
                 <configuration>
                   <dataFileIncludes>
                     <dataFileInclude>**/jacoco.exec</dataFileInclude>
+                    <dataFileInclude>**/jacoco-quarkus.exec</dataFileInclude>
```

Comment:
The Quarkus extension just writes `jacoco.exec`, do we need this one?

Reply:
https://quarkus.io/guides/tests-with-coverage#running-the-tests-with-coverage it actually doesn't. We can change it to write `jacoco.exec` if this test works though, and remove this line.

Reply:
Ah, right.